beacon-chain/params/config.go

```diff
@@ -10,53 +10,59 @@ var env = "default"
 
 // Config contains configs for node to participate in beacon chain.
 type Config struct {
-	ShardCount                  int       // ShardCount is a fixed number.
-	DefaultBalance              *big.Int  // DefaultBalance of a validator in wei.
-	MaxValidators               int       // MaxValidators in the protocol.
-	Cofactor                    int       // Cofactor is used cutoff algorithm to select slot and shard cutoffs.
+	ShardCount                  int       // ShardCount is the fixed number of shards in Ethereum 2.0.
+	DepositSize                 *big.Int  // DepositSize is how much a validator has deposited in wei.
 	BootstrappedValidatorsCount int       // BootstrappedValidatorsCount is the number of validators we seed the first crystallized state.
+	MaxValidators               int       // MaxValidators is the max number of validators allowed in Ethereum 2.0.
 	EtherDenomination           int       // EtherDenomination is the denomination of ether in wei.
-	CycleLength                 uint64    // CycleLength is the beacon chain cycle length in slots.
-	SlotDuration                uint64    // SlotDuration in seconds.
+	CycleLength                 uint64    // CycleLength is one beacon chain cycle length in slots.
+	SlotDuration                uint64    // SlotDuration is how many seconds are in a single slot.
 	MinCommiteeSize             uint64    // MinDynastyLength is the slots needed before dynasty transition happens.
 	DefaultEndDynasty           uint64    // DefaultEndDynasty is the upper bound of dynasty. We use it to track queued and exited validators.
 	MinDynastyLength            uint64    // MinCommiteeSize is the minimal number of validator needs to be in a committee.
 	BaseRewardQuotient          uint64    // BaseRewardQuotient is used to calculate validator per-slot interest rate.
-	SqrtDropTime                uint64    // SqrtDropTime is a constant to reflect time it takes to cut offline validators’ deposits by 39.4%.
+	SqrtEDropTime               uint64    // SqrtEDropTime is a constant to reflect time it takes to cut offline validators’ deposits by 39.4%.
```

Comment:
What does it mean though? What does the E stand for?

Reply:
E is this guys 
https://en.wikipedia.org/wiki/E_(mathematical_constant)

Reply:
Rename to Exp perhaps as that is more standard in computing